### DISCUSSION TITLE
edit button not showing up

### DISCUSSION THREAD

**Discussion by romeroj1** [McShelby/hugo-theme-relearn#759] [Q&A]
I have set the editURL: <urlvalue> in my config but the edit button still doesn't show up. What am I missing?

**Comment by McShelby** [comment]
This works as you can see in the exampleSite.

More information is needed. At best a link to your repo.

**Reply by pete-lambda** [reply]
Hi, I'm having the same issue. I added the editUrl parameter to my hugo.toml file, and the edit icon didn't appear in the top bar. So I created a config/params.toml file and added the editUrl there, but it still doesn't appear. My repo is https://github.com/pete-lambda/lambda-hugo-poc thank you!

**Reply by McShelby** [reply]
Moving your `config/params.toml` to `config/_default/params.toml` should do the trick

**Reply by pete-lambda** [reply]
Beautiful, that did the trick, thank you! Sorry I missed the second level directory.

**Reply by McShelby** [reply]
Great that this solved your issue. If you are pleased with the theme, I would be more then happy if you would give it a star.

**Reply by pete-lambda** [reply]
Done! Thank you for working on this!
